.docker/drupal/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM alpine:3.8
+
+RUN apk add php sqlite composer git \
+    php7-pdo php7-sqlite3 php7-pdo_sqlite php7-gd php7-openssl php7-json \
+    php7-mbstring php7-dom php7-session php7-xml php7-simplexml php7-tokenizer \
+    php7-curl php7-xmlwriter php7-ctype \
+    php-apache2 apache2 apache2-ctl
+
+RUN rm -rf /var/www/localhost && \
+    sed -i 's/\/var\/www\/localhost\/htdocs/\/var\/www\/drupal\/docroot/g' /etc/apache2/httpd.conf && \
```

Comment:
I think it would be nice to enable opcache in the php ini

Reply:
:+1: